### PR TITLE
52350_Edit product

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52350)

## WHAT (optional)
- Edit product

## HOW
- similar to create product function

## Evidence (Screenshot or Video)
Screenshot from 2022-08-18 16-48-12

### CODE REVIEW

**Review by diemntn-0165 — APPROVED** [review]

**Comment by fonqhoanq** [comment]
ready

**Comment by fonqhoanq** [comment]
ready

File: app/controllers/admin/products_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Admin::ProductsController < Admin::BaseController
+  before_action :find_product_by_id, except: [:index, :new, :create]
```
Comment (ngocvt-0484) [inline app/controllers/admin/products_controller.rb]: thêm phía dưới dòng trống

Comment (ngocvt-0484) [inline app/controllers/admin/products_controller.rb]: dùng %i thay cho []

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**